src/utils/utils.ts

```diff
@@ -320,6 +320,18 @@ export function typeNameFromTypeNode(node: TypeNode, ast: AST): TypeName {
   return result;
 }
 
+export function getContractTypeString(node: ContractDefinition): string {
```

Comment:
I think with the number of typestring generating functions we're creating, it would be best to have a separate file in the utils folder for all of them